releng/com.mimacom.ddd.repository/com.mimacom.ddd.product

```diff
@@ -35,6 +35,8 @@
       <feature id="com.mimacom.ddd.dm.esm.feature" installMode="root"/>
       <feature id="com.mimacom.ddd.sm.asm.feature" installMode="root"/>
       <feature id="com.mimacom.ddd.sm.sim.feature" installMode="root"/>
+      <feature id="com.mimacom.ddd.pub.proto.feature" installMode="root"/>
+      <feature id="com.mimacom.ddd.pub.pub.feature" installMode="root"/>
```

Comment:
I think these features were successfully included in the product:

<img width="805" alt="image" src="https://user-images.githubusercontent.com/1107810/79303445-359d3480-7eef-11ea-8b96-f7e91d91afd3.png">